PointOfInteraction/SmartPointOfInteraction/doc/spec.md

```diff
@@ -0,0 +1,71 @@
+# SmartPointOfInteraction
+
+## Description
+
+A Smart Point of Interaction defines a place with technology to interact with citizens, for example, through Beacon technology from Apple, Eddystone/Physical-Web from Google or other proximity-based interfaces. Since the interactive area could be composed by more than one device providing the technology, this model represents a group of Smart Spot devices.
+
+The data model includes information regarding the area/surface covered by the technology (i.e., the area covered by Bluetooth Low Energy-based Beacon or Eddystone), the schedule of the broadcasted information, links to the multimedia resources / Web Apps, etc. In addition, the data model may have a reference to another OMA NGSI entity such as a Parking, a Point of Interest (POI), etc. with enriched interaction provided by this technology.
+
+This entity is purely virtual, is not a device mapping.
+
+## Data Model
+
++ `id` : Unique identifier. 
+
++ `type` : Entity type. It must be equal to `SmartPointOfInteraction`.
+
++ `category` : Defines the type of interaction.
+    + Attribute type: List of [Text](http://schema.org/Text)
+    + Allowed values: `information`, `entertainment`, `infotainment` or `co-creation`.
+    + Mandatory
+    
++ `areaCovered` : Defines the area covered by the Smart Point of Interaction using geoJSON format.
+    + Attribute type: `geo:json`.
+    + Normative References: [https://tools.ietf.org/html/rfc7946](https://tools.ietf.org/html/rfc7946)
+    + Optional    
+    
++ `applicationUrl` : This field specifies the real URL containing the solution or application (information, co-creation, etc) while the SmartSport 'announcedUrl' field specifies the broadcasted URL which could be this same URL or a shortened URL.
+    + Attribute type: [URL](https://schema.org/URL)
+    + Mandatory    
+
++ `availability`: Specifies the time intervals in which this service is available. This is a general service information while SmartSpots have their own availability in order to allow advanced configurations.
+    + Attribute type: [openingHours](https://schema.org/openingHours)
```

Comment:
`openingHours` is not an attribute type, it is a property. You have to say here that the property value format is the same as  the one for `openingHours` in schema.org. See for instance the property `permitActiveHours` of http://fiware-datamodels.readthedocs.io/en/latest/Parking/ParkingGroup/doc/spec/index.html